webservices/server-integration/src/main/java/org/jboss/as/webservices/deployers/WSLibraryFilterProcessor.java

```diff
@@ -66,6 +65,13 @@ public void deploy(final DeploymentPhaseContext phaseContext) throws DeploymentU
                 AttachmentList<ResourceRoot> resourceRoots = unit.getAttachment(RESOURCE_ROOTS);
                 if (resourceRoots != null) {
                     for (ResourceRoot root : resourceRoots) {
+                        if (root.getRoot().getChild("META-INF/services/javax.xml.datatype.DatatypeFactory").exists()
+                                || root.getRoot().getChild("META-INF/services/javax.xml.parsers.DocumentBuilderFactory").exists()
+                                || root.getRoot().getChild("META-INF/services/javax.xml.parsers.SAXParserFactory").exists()
+                                || root.getRoot().getChild("META-INF/services/jjavax.xml.validation.SchemaFactory").exists()) {
```

Comment:
I think you mean `javax.xml.validation.SchemaFactory`?

Reply:
@pferraro  Yes. Thanks for review. Already fixed.